SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1281,9 +1280,20 @@ void USpatialActorChannel::UpdateSpatialPosition()
 
 void USpatialActorChannel::SendPositionUpdate(AActor* InActor, Worker_EntityId InEntityId, const FVector& NewPosition)
 {
-	if (InEntityId != SpatialConstants::INVALID_ENTITY_ID && NetDriver->HasServerAuthority(InEntityId))
+	if (InEntityId != SpatialConstants::INVALID_ENTITY_ID)
 	{
-		Sender->SendPositionUpdate(InEntityId, NewPosition);
+#if !UE_BUILD_SHIPPING
+		if (!NetDriver->Connection->GetView()[EntityId].Authority.Contains(SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID))
+		{
+			UE_LOG(LogSpatialSender, Verbose,
```

Comment:
Log category cleanup?

Reply:
Great catch, wow, for some reason I thought this wouldn't compile so that would catch these. removed spatial sender log category and changed all these